acceptance/features/user-crds.feature

```diff
@@ -18,59 +18,69 @@ Feature: User CRDs
     And "alice" should exist and be able to authenticate to the "sasl" cluster
 
   @skip:gke @skip:aks @skip:eks
-  Scenario: Managing Authentication-only Users
+  Scenario: Manage authentication-only users
     Given there is no user "jason" in cluster "sasl"
     And there are already the following ACLs in cluster "sasl":
       | user   | acls |
       | jason  | [{"type":"allow","resource":{"type":"cluster"},"operations":["Read"]}] |
     When I apply Kubernetes manifest:
     """
+# tag::manage-authn-only-manifest[]
+# In this example manifest, a user called "jason" is created in a cluster called "sasl".
+# The user's password is defined in a Secret called "jason-password".
```

Comment:
Can this sentence be more specific as behavior would be different based on the Secret created apriori the User custom resource.

1. If Secret exist and key exist within that Secret that password would be used.
2. If Secret exist but key does not exist then Secret is updated with autogenerated password.
3. If Secret does not exist Secret with that key is created with autogenerated password.

@andrewstucki Maybe we should create more test cases to expose/show behavior?

Reply:
oh, nice! I didn't know about 2 and 3 👀